### DISCUSSION TITLE
Julia API should be able to execute multiple statements in one API call

### DISCUSSION THREAD

**Discussion by slwu89** [duckdb/duckdb#15264] [Ideas]
Similar to https://github.com/duckdb/duckdb/issues/400, it seems that the Julia API does not currently support executing multiple statements in one API call. For example, following the test (but translating to Julia) in https://github.com/duckdb/duckdb/pull/1102/files#diff-f29a8dfe777bc975a81ecf7c8196e71aec926226ff40c7fc1cfec6e684382a06, on `[d2f5444f] DuckDB v1.1.0`, I get the error `ERROR: Invalid Input Error: Cannot prepare multiple statements at once!`

```julia
using DuckDB
con = DBInterface.connect(DuckDB.DB, ":memory:")

DBInterface.execute(con, """
    CREATE TABLE integers(i integer);
    insert into integers select * from range(10);
    select * from integers;
""")
```

Changing `DBInterface.execute` to `DBInterface.executemultiple` results in the same error.

**Comment by slwu89** [comment]
Ah, I see now that `DuckDB.query` allows multiple statements, but the DBInterface.jl package has trouble with it https://juliadatabases.org/DBInterface.jl/dev/#DBInterface.executemultiple.

I'm new to the Julia Database ecosystem so I'm unsure if this is something that should be raised to whoever is responsible for DBInterface.jl

**Reply by schlichtanders** [reply]
@slwu89 any progress?

**Reply by slwu89** [reply]
No, unfortunately for the time being I've gone back to Python for my DB interaction needs.
